src/utils/init.d.ts

```diff
@@ -0,0 +1,31 @@
+export function init(
+  apiKey: string,
+  options?: {
+    userKey?: null | string;
+    formKeys?: string[];
+    tracking?: 'cookie' | 'fingerprint';
+    authId?: string;
+    authEmail?: string;
+    authPhoneNumber?: string;
+  }
+  // not totally sure about this return type - <void> is the current implicit type
```

Comment:
another question

Reply:
Void seems right to me.  The docs make no "promises" on nor mention of the return type.

Reply:
if you look at the init definition, it returns an awaitable promise that returns void as you specified here. if the docs don't mention it could you go in and add it?

Reply:
specifying the awaitability here IMO is important

Reply:
We do say that it is an async function function in the docs